test/e2e/util.go

```diff
@@ -255,3 +260,20 @@ func equalFQDN(name1, name2 string) bool {
 	}
 	return name1 == name2
 }
+
+func newHostNetworkController(name types.NamespacedName, domain string) *operatorv1.IngressController {
+	repl := int32(1)
+	return &operatorv1.IngressController{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: name.Namespace,
+			Name:      name.Name,
+		},
+		Spec: operatorv1.IngressControllerSpec{
+			Domain:   domain,
+			Replicas: &repl,
```

Comment:
You can use the [pointer helper function](https://pkg.go.dev/k8s.io/utils/pointer#Int32) here.

Reply:
done